package.json

```diff
@@ -20,13 +20,15 @@
     "@jsonforms/core": "^2.5.2",
     "@jsonforms/material-renderers": "^2.5.2",
     "@jsonforms/react": "^2.5.2",
+    "@material-ui/core": "^4.12.4",
```

Comment:
Do you have some future plans for this? Because it doesn't seem to be used anywhere in the project. I think the "@material-ui/*" imports are for the old version of the library, which we don't want to use if there's any way around it.